docs/learn-staking.md

```diff
@@ -158,11 +156,11 @@ Validators can create a cut of the reward that is not shared with the nominators
 For example, assume reward is 100 DOTs.
 A validator may specify `validator_payment = 50 DOTs` and the remaining 50 DOTs would be split between the validator and their nominators based on the portion of stakes they had.
 
-Rewards can be used by the same account (controller) to keep accumulating the rewards or by the stash account (increasing the staked value/not increasing the staked value). Also, it is possible to top-up/withdraw partial bonded DOTs without having to completely un-stake everything.
+Rewards can be used by the same account (controller) to keep accumulating the rewards or by the stash account (increasing the staked value / not increasing the staked value). Also, it is possible to top-up / withdraw some bonded DOTs without having to un-stake everything.
 
 ## Inflation
 
-It is planned to be close to 10% in the first year. This means that each validator will get 1,000 - 2,000 DOTs per month to share with their nominators.
+Planned to be close to 10% in the first year. This means that each validator will get 1,000 - 2,000 DOTs per month to share with their nominators.
```

Comment:
Why do you think we should change this from a sentence to a sentence fragment?  I think even better might be:

```suggestion
Inflation is planned to be close to 10% in the first year. This means that each validator will get 1,000 - 2,000 DOTs per month to share with their nominators.
```

Reply:
Inflation is planned sounds weird to me, just personal preference.

Reply:
> Inflation is _designed_ to target 10%

is usually how I phrase it

Reply:
Sounds better to me

Reply:
```suggestion
Inflation is designed to be close to 10% in the first year. This means that each validator will get 1,000 - 2,000 DOTs per month to share with their nominators.
```